src/components/ShoeDetails.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="mt-12">
+    <p class="text-h4 font-weight-black">Nike Shoe Name 15</p>
+    <p class="text-h5 font-weight-medium text-grey-darken-2 mt-2">₱ 5,000</p>
+  </div>
+  <div class="mt-6">
+    <p class="text-body-1 text-grey-darken-1">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+      commodo consequat.
+    </p>
+    <p class="text-body-2 font-weight-bold mt-3">Men's Basketball Shoes</p>
+  </div>
+</template>
+
+<script setup />
```

Comment:
@balanejacob You can just remove the setup script if it's empty.

Reply:
@balanejacob The shoe details in this component, define them as variables in the parent component and just pass them as props to this component.